libr/include/r_io.h

```diff
@@ -289,6 +290,7 @@ typedef struct r_io_bind_t {
 	RIOAlFree al_free;	//needed for esil
 	RIOAlGetFbufByflags al_buf_byflags;	//needed for esil
 	RIOIsValidOff is_valid_offset;
+	RIOAddrIsMapped addr_is_mapped;
```

Comment:
I'd call it `is_addr_mapped`.

Reply:
`addr_is_mapped` has the same suffix of `r_io_addr_is_mapped()` though

Reply:
Honestly I'd rename that as well eheh. But fair point :)

I think `is_addr_mapped` is better (like `is_valid_offset`) but if that's already like this, then :+1